src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +13,8 @@ public class Users {
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        return users.stream()
+                .filter( u -> u.matchName(name))
+                .findAny().orElse(DEFAULT_USER);
```

Comment:
```suggestion
                .findAny()
                .orElse(DEFAULT_USER);
```

저는 개인적이론 `stream` 사용할때 한줄당 하나씩 하는편이에요.
가독성과 디버깅에 이점이 있는것 같아서요! 😄